packages/flutter/test/material/snack_bar_test.dart

```diff
@@ -446,39 +452,84 @@ void main() {
         ),
       ),
     ));
-    expect(find.text('bar1'), findsNothing);
-    expect(find.text('bar2'), findsNothing);
-    await tester.tap(find.byKey(tapTarget)); // queue bar1
-    await tester.tap(find.byKey(tapTarget)); // queue bar2
-    expect(find.text('bar1'), findsNothing);
-    expect(find.text('bar2'), findsNothing);
-    await tester.pump(); // schedule animation for bar1
-    expect(find.text('bar1'), findsOneWidget);
-    expect(find.text('bar2'), findsNothing);
-    await tester.pump(); // begin animation
-    expect(find.text('bar1'), findsOneWidget);
-    expect(find.text('bar2'), findsNothing);
-    await tester.pump(const Duration(milliseconds: 750)); // 0.75s // animation last frame; two second timer starts here
-    await tester.drag(find.text('bar1'), const Offset(0.0, 50.0));
-    await tester.pump(); // bar1 dismissed, bar2 begins animating
-    expect(find.text('bar1'), findsNothing);
-    expect(find.text('bar2'), findsOneWidget);
+
+    final Map<int, Offset> dragGestures = <int, Offset>{};
```

Comment:
This is some pretty nifty test craft! Can you add some comments along the way explaining what is happening? Also, since there are two of these, we might want to roll some of the duplicated code up to be re-used.

Reply:
@Piinks Thanks! I already updated the test and simplified the drag gestures creation to be self-explanatory. a612ed7